tests/unittest_inference.py

```diff
@@ -4050,6 +4050,42 @@ def inner():
         assert not isinstance(inferred, nodes.ClassDef)  # was inferred as builtins.type
         assert inferred is util.Uninferable
 
+    def test_infer_subclass_attr_instance_attr_indirect(self):
+        node = extract_node(
+            """
+        class Parent:
+            def __init__(self):
+                self.data = 123
+
+        class Test(Parent):
+            pass
+        t = Test()
+        t
+        """
+        )
+        inferred = next(node.infer())
+        # self.assertIsInstance(inferred, bases.Instance)
```

Comment:
Why comment it out? Either test or remove it. The test should be fine

Reply:
Just cruft I overlooked